### ISSUE TITLE
NLAD API on command

### ISSUE THREAD

**Issue by shivakrishna-vcare** [shivakrishna-vcare/OrangeScrum#69]
<p>&nbsp;</p>
<p>Request the addition of permission based NLAD Enroll and NLAD Transfer buttons to the Telgoo5 CSR. These two buttons shall be located under Customer Service (PC400), Customer History (PC226) next to the NLAD HISTORY label. These buttons would force another enroll or transfer API query to be processed and sent to NLAD.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending generic/CSR/common/redirect.php<br />Sending generic/CSR/history/history.php<br />Adding generic/CSR/history/history_ajax.php<br />Transmitting file data ...done<br />Committing transaction...<br />Committed revision 66271.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66271 &gt; QA 66272 &gt; DEMO 66273</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending generic/CSR/history/history.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 66287.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66287 &gt; QA 66288 &gt; DEMO 66289</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending generic/CSR/history/history_ajax.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 66294.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66294 &gt; QA 66296 &gt; DEMO 66300&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending generic/CSR/history/history.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 66302.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66302 &gt; QA 66307 &gt; DEMO 66308</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending generic/CSR/history/history.php<br />Sending generic/CSR/history/history_ajax.php<br />Transmitting file data ..done<br />Committing transaction...<br />Committed revision 66344.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66344 &gt;QA 66345 &gt;Demo 66346</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Pushkar Sir,&nbsp;</p>
<p>This task is verified and tested on demo, now you can move to live after client confirmation.&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>66271 &gt; QA 66272 &gt; DEMO 66273 &gt; Production<br />66344 &gt; QA 66345 &gt; DEMO 66346 &gt; Production<br /><br /></p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66271 &gt; QA 66272 &gt; DEMO 66273 &gt; Production 66863<br />66344 &gt; QA 66345 &gt; DEMO 66346 &gt; Production updated</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending history.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 66973.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66973 &gt; QA 66974 &gt; DEMO 66975</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Now allowing for disconnected&nbsp; customer.&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>66973 &gt; QA 66974 &gt; DEMO 66975 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>66973 &gt; QA 66974 &gt; DEMO 66975 &gt; Production 66979</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Closing this task.&nbsp;</p>
